### PR TITLE
[ONNX] CentOS disable unit tests

### PR DESCRIPTION
**Pull request by apwojcik**
We have issues with Protobuf in CentOS. This PR temporarily disables unit testing of ONNX for CentOS until we resolve the issues. Unit tests for Ubuntu are not affected.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -101,6 +101,10 @@ endif()
 # Installation logic...
 #-----------------------------------------------------------------------------------------------
 
+if (NOT APPLE AND NOT WIN32)
```
Comment (diyessi) [inline CMakeLists.txt]: Please select which systems you want, not which you don't want.

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -47,7 +47,15 @@ set(SRC
 )
 
 if (NGRAPH_ONNX_IMPORT_ENABLE)
-    list(APPEND SRC onnx_import.cpp)
+    if (APPLE OR WIN32)
+        list(APPEND SRC onnx_import.cpp)
+    else()
```
Comment (diyessi) [inline test/CMakeLists.txt]: Change else to if linux.

Comment (apwojcik) [reply test/CMakeLists.txt]: There is no `if(LINUX)` in CMake. Only `if(UNIX)`, but is set for both Linux and Apple.

File: CMakeLists.txt (hunk 1 above)
Comment (apwojcik) [reply CMakeLists.txt]: There is no `if(LINUX)` in CMake. Only `if(UNIX)`, but is set for both Linux and Apple.

Comment (diyessi) [reply CMakeLists.txt]: No wonder no one used CMAKE back in the old days.
`if (unix and not apple)` isn't any better.

**Pull request merged**